app/src/services/object.js

```diff
@@ -76,6 +76,21 @@ const service = {
     }
   },
 
+  /**
+   * @function searchMetadata
+    * Search and filter for specific metadata keys
+   * @param {object} [params.metadata] Optional object of metadata keys to filter on
+   * @returns {Promise<object[]>} The result of running the find operation
+   */
+  searchMetadata: (params) => {
+    return Metadata.query()
+      .modify('filterKeyValue', { metadata: params.metadata })
+      .then(result => result.map(row => {
+        // eslint-disable-next-line no-unused-vars
```

Comment:
just checking if you need this linting exception thing

Reply:
Removed it.